Stub missing for os.supports_dir_fd, os.supports_fd, os.supports_effective_ids, os.supports_follow_symlinks

Stubs seem to be missing for the listed `set` instances in the `os` module.

```
 $ mypy --version
mypy 0.501
 $ cat test.py
import os
os.symlink in os.supports_dir_fd
os.chdir in os.supports_fd
os.supports_bytes_environ
os.access in os.supports_effective_ids
os.stat in os.supports_follow_symlinks
os.path.supports_unicode_filenames

 $ mypy test.py
test.py:6: error: "module" has no attribute "supports_dir_fd"
test.py:7: error: "module" has no attribute "supports_fd"
test.py:9: error: "module" has no attribute "supports_effective_ids"
test.py:10: error: "module" has no attribute "supports_follow_symlinks"
```

--- a/stdlib/3/os/__init__.pyi
+++ b/stdlib/3/os/__init__.pyi
@@ -3,7 +3,7 @@
 
 # based on http: //docs.python.org/3.2/library/os.html
 
-from builtins import function, OSError as error
+from builtins import OSError as error
 from io import TextIOWrapper as _TextIOWrapper
 import sys
 from typing import (
@@ -18,10 +18,10 @@ from mypy_extensions import NoReturn
 supports_bytes_environ = False  # TODO: True when bytes implemented?
 
 if sys.version_info >= (3, 3):
-    supports_dir_fd = ...  # type: Set[function]
-    supports_fd = ...  # type: Set[function]
-    supports_effective_ids = ...  # type: Set[function]
-    supports_follow_symlinks = ...  # type: Set[function]
+    supports_dir_fd = ...  # type: Set[Callable[..., Any]]
+    supports_fd = ...  # type: Set[Callable[..., Any]]
+    supports_effective_ids = ...  # type: Set[Callable[..., Any]]
+    supports_follow_symlinks = ...  # type: Set[Callable[..., Any]]
 
 SEEK_SET = 0
 SEEK_CUR = 0

--- a/stdlib/3/os/__init__.pyi
+++ b/stdlib/3/os/__init__.pyi
@@ -3,7 +3,7 @@
 
 # based on http: //docs.python.org/3.2/library/os.html
 
-from builtins import OSError as error
+from builtins import function, OSError as error
 from io import TextIOWrapper as _TextIOWrapper
 import sys
 from typing import (
@@ -16,6 +16,12 @@ from mypy_extensions import NoReturn
 # ----- os variables -----
 
 supports_bytes_environ = False  # TODO: True when bytes implemented?
+
+if sys.version_info >= (3, 3):
+    supports_dir_fd = ...  # type: Set[function]
+    supports_fd = ...  # type: Set[function]
+    supports_effective_ids = ...  # type: Set[function]
+    supports_follow_symlinks = ...  # type: Set[function]
 
 SEEK_SET = 0
 SEEK_CUR = 0

